src/languages/llvm.js

```diff
@@ -109,12 +109,11 @@ export default function(hljs) {
       // another language than an actual comment
       hljs.COMMENT(/;\s*$/, null, { relevance: 0 }),
       hljs.COMMENT(/;/, /$/),
-      hljs.QUOTE_STRING_MODE,
       {
         className: 'string',
         variants: [
           // Double-quoted string
-          { begin: /"/, end: /[^\\]"/ },
+          { begin: /"/, end: /"/ },
```

Comment:
If there is only one rules now then we no longer need the `variants` parent... and do you think while you're here perhaps we could add the actual escaping.  Is it not `\00` style, if I'm not mistaken?  IE, you'd just need a child inside `contains` that matched `\\\d\d` and scoped as `char.escape`.

Thoughts?